src/administration/integrations/gitlab.md

```diff
@@ -33,7 +33,7 @@ Note that only `project owner` or `project admin` can manage the integrations.
 Open a terminal window (you need to have the Platform.sh CLI installed). Enable the GitLab integration as follows:
 
 ```bash
-platform integration:add --type=gitlab --token=THE-TOKEN-YOU-WROTE-DOWN --base-url=https://THE-URL-OF-YOUR-GITLAB --gitlab-project=MY-NAMESPACE/MY-PROJECTNAME
+platform integration:add --type=gitlab --token=GITLAB-ACCESS-OKEN --base-url=https://THE-URL-OF-YOUR-GITLAB --server-project=MY-NAMESPACE/MY-PROJECTNAME --project=PROJECT_ID
```

Comment:
Hey Chad, what do you think we make it even more explicit. Such as `PLATFORMSH_PROJECT_ID` the same thing to the GitHub documentation.